docs/csharp/misc/cs0162.md

```diff
@@ -37,3 +37,26 @@ public class Program
     }
 }
 ```
+
+Another common example where this error is generated is as follows:
+
+```csharp
+public static class Class1
+{
+    public static string Method1()
+    {
+        string x = "a";
+        switch (x)
+        {
+            case "a":
+                return "a";
+                break;          // CS0162
+        }
+        return "";
+    }
+}
+```
+
+Even though the `break` statements is part of code formatting, the
+fact that a `return` is issued prior to reaching the `break`
+statement, results in it becoming unreachable.
```

Comment:
I'll suggest the following edit:

```suggestion
The `break` statement cannot be reached because it occurs after the `return` statement.
The `return` statement ends the enclosing `case` branch.
```